HubSpot.NET/Api/Contact/Dto/ContactHubSpotModel.cs

```diff
@@ -11,49 +14,69 @@ namespace HubSpot.NET.Api.Contact.Dto
     [DataContract]
     public class ContactHubSpotModel : IHubSpotModel
     {
+        public ContactHubSpotModel() { }
+
         /// <summary>
         /// Contacts unique ID in HubSpot
         /// </summary>
         [DataMember(Name = "vid")]
         [IgnoreDataMember]
         public long? Id { get; set; }
+        
         [DataMember(Name = "email")]
         public string Email { get; set; }
+        
         [DataMember(Name = "firstname")]
         public string FirstName { get; set; }
+        
         [DataMember(Name = "lastname")]
         public string LastName { get; set; }
+        
         [DataMember(Name = "website")]
         public string Website { get; set; }
         [DataMember(Name = "company")]
-        public string Company { get; set; }
+        public string Company { set; get; }
         [DataMember(Name = "phone")]
-        public string Phone { get; set; }
+        public string Phone { set; get; }
         [DataMember(Name = "address")]
-        public string Address { get; set; }
+        public string Address { set; get; }
         [DataMember(Name = "city")]
-        public string City { get; set; }
+        public string City { set; get; }
         [DataMember(Name = "state")]
-        public string State { get; set; }
+        public string State { set; get; }
         [DataMember(Name = "zip")]
-        public string ZipCode { get; set; }
+        public string ZipCode { set; get; }
 
-        [DataMember(Name="associatedcompanyid")]
-        public long? AssociatedCompanyId { get;set; }
+        [DataMember(Name = "associatedcompanyid")]
+        public long? AssociatedCompanyId { get; set; }
 
-        [DataMember(Name="hubspot_owner_id")]
-        public long? OwnerId { get;set; }
+        [DataMember(Name = "hubspot_owner_id")]
+        public long? OwnerId { get; set; }
 
-        public string RouteBasePath => "/contacts/v1";
-        public bool IsNameValue => false;
-        public virtual void ToHubSpotDataEntity(ref dynamic converted)
+        // Used for return values
+        [DataMember(Name = "properties")]
+        public Dictionary<string, ContactProperty> Properties { get; set; } = new Dictionary<string, ContactProperty>();
+
+        /// <summary>
+        /// Loads all properties into the ContactProperty Dictionary even from custom contact models
+        /// </summary>
+        public void LoadProperties()
         {
+            PropertyInfo[] properties = GetType().GetProperties();
```

Comment:
Does this get called automatically at transport time?

Reply:
I can't remember the exact steps at the moment. I would need to run it again.